horace_core/algorithms/change_crystal_dnd.m

```diff
@@ -1,42 +1,52 @@
-function change_crystal_dnd(varargin)
-% Change the crystal lattice and orientation in a file or set of files containing dnd information
-% 
-% Most commonly:
-%   >> change_crystal (file, rlu_corr)              % change lattice parameters and orientation
+function varargout = change_crystal_dnd(varargin)
```

Comment:
given that all the usage examples start off with in_data, alignment_info, why are these arguments not explicit here? They are explicit in change_crystal_sqw